semmc-aarch32/src/SemMC/Architecture/ARM/UF.hs

```diff
@@ -42,6 +40,8 @@ type BaseIdxType arch = BaseBVType (A.RegWidth arch)
 --type BaseMemType arch = BaseArrayType (SingleCtx (BaseIdxType arch)) (BaseBVType 8)
 type BaseMemType arch = G.MemoryBaseType
 
+type UnitType = BaseBoolType
```

Comment:
Can we get a comment as to why Unit~Bool?  If it is just because there is no unit in what4, it might be clearer to use the isomorphic `BaseStructType EmptyCtx`